src/test/java/giis/eshopcontainers/e2e/functional/common/BrowserUser.java

```diff
@@ -0,0 +1,68 @@
+package giis.eshopcontainers.e2e.functional.common;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+
+public class BrowserUser {
+    private static final Logger log= LoggerFactory.getLogger(BaseLoggedTest.class);
+    protected final String clientData;
+    protected final int timeOfWaitInSeconds;
+    protected WebDriver driver;
+    protected boolean isOnSession;
+    protected WebDriverWait waiter;
+
+    public BrowserUser(String clientData, int timeOfWaitInSeconds) {
```

Comment:
It would be interesting to use this tool, especially since it's maintained/created by you. However, it might be more appropriate to create another pull request for this feature No?

Reply:
Still working on instantiate the driver with selema @javiertuya. 
The rest of the changes are done.

Reply:
You decide, it was only a suggestion, it was only a suggestion to simplify driver creation